src/components/DatePickerComponent.web.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import DateFnsUtils from "@date-io/date-fns";
+import {
+  MuiPickersUtilsProvider,
+  DatePicker,
+  TimePicker,
+  DateTimePicker,
+} from "@material-ui/pickers";
+
+interface Props {
```

Comment:
are the props between web and native the same? can we share those inside the main DatePicker file?

Reply:
that would lead to both DatePicker and DatePickerComponent to be importing each other which causes issues. I'll put it in a separate file and put them all in a folder, feel that's most appropriate.